### ISSUE TITLE
Out of game save notification sound

### ISSUE THREAD

**Issue by WatislavB** [EZ64cool/obs-hadowplay#19]
I use Replay Buffer all the time, even on the desktop, since auto-enabling may not always work, as happened with the game The Finals, due to the fact that it is not captured by game_capture. In addition, I often use the Replay Buffer just on the desktop, but in this case, if obs_hadowplay does not capture the game, it does not play the save notification sound. Wouldn't it be more logical to always play the Replay Buffer save notification sound, regardless of whether obs_hadowplay has detected the game? Or at least it would be possible to make such a setting in the plugin settings window? This would completely replace the obsnotification script for me.

Thank you ❤️

**Comment by EZ64cool** [comment]
Yeah there's no reason I couldn't allow this
Currently it works only when the replay has been "re-organised" into a named folder, but it probably makes more sense to have it play for any save/make it an option.

**Comment by ipsusu** [comment]
+1 to this, would be awesome. still using a notification script in addition to the plugin right now because i need to know if my clip is saved correctly on games where it doesn't properly switch to my game mode scene and falls back on my desktop capture scene.

perhaps something i can fix but idk how, so this would be very helpful.

**Comment by WatislavB** [comment]
> +1 to this, would be awesome. still using a notification script in addition to the plugin right now because i need to know if my clip is saved correctly on games where it doesn't properly switch to my game capture scene (supposed to auto switch when a fullscreen app is focused) and falls back on my desktop capture scene.
> 
> perhaps something i can fix but idk how, so this would be very helpful.

To automatically switch scenes, I use the Advanced Scene Switcher plugin. I created 2 macros that switch the scenes for the game capture and the desktop (the desktop capture scene still needs to have a game capture source to see that the game is running). In the macro conditions, the Window is checked for "Window is fullscreen", "Window is focused" and the Game Capture source is checked for "change". In this case, the macro switches the scene to capture the game without capturing the screen. Likewise, the second macro changes the scene to capture the desktop if I minimize or close the game

<details>

![image](https://github.com/user-attachments/assets/f18eed94-3805-41b0-a17c-948a95c65f0c)

</details>

<details>

![image](https://github.com/user-attachments/assets/90d563e5-216f-41c2-8781-346759904f11)

</details>

<details>

![image](https://github.com/user-attachments/assets/6a889f7b-bccd-490f-8022-b7b65fe18b32)

</details>

However, if the game is not running, but the Replay Buffer is running to record the desktop - I would also like to see notifications when saving

**Comment by ipsusu** [comment]
> > +1 to this, would be awesome. still using a notification script in addition to the plugin right now because i need to know if my clip is saved correctly on games where it doesn't properly switch to my game capture scene (supposed to auto switch when a fullscreen app is focused) and falls back on my desktop capture scene.
> > perhaps something i can fix but idk how, so this would be very helpful.
> 
> To automatically switch scenes, I use the Advanced Scene Switcher plugin. I created 2 macros that switch the scenes for the game capture and the desktop (the desktop capture scene still needs to have a game capture source to see that the game is running). In the macro conditions, the Window is checked for "Window is fullscreen", "Window is focused" and the Game Capture source is checked for "change". In this case, the macro switches the scene to capture the game without capturing the screen. Likewise, the second macro changes the scene to capture the desktop if I minimize or close the game
> 
> ![image](https://private-user-images.githubusercontent.com/91764346/350725685-f18eed94-3805-41b0-a17c-948a95c65f0c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE1MjQwNzYsIm5iZiI6MTcyMTUyMzc3NiwicGF0aCI6Ii85MTc2NDM0Ni8zNTA3MjU2ODUtZjE4ZWVkOTQtMzgwNS00MWIwLWExN2MtOTQ4YTk1YzY1ZjBjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzIxVDAxMDI1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVkZDNkZTIxZDAxMjczNzE1MmQ4OTY2NmM0ODMyOTk0ZDdhOGY4NDk1NWUyNjFhMTA5MmVkYzU0ZjFjN2FlMjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qXYXYQ7Jv9h2NWUs4dhhfEkbi8TM3VPba3NBudZ2Gnk)
> 
> However, if the game is not running, but the Replay Buffer is running to record the desktop - I would also like to see notifications when saving

yes! i do the exact same, but for whatever reason OBS doesn't detect certain games as fullscreen (possibly due to overlays etc? it's a fullscreen borderless window). even if i make it switch via a hardcoded window name in the macro, the game capture still doesn't work.

perhaps i could have a specific game capture scene for each of these titles i identify, and hardcode it to switch? would make stuff more complicated but oh well...

**Comment by EZ64cool** [comment]
This should now be fixed in version 2.0.4
